### DISCUSSION TITLE
Is it possible to change multiple items in once?

### DISCUSSION THREAD

**Discussion by yellowjs0304** [wkentaro/labelme#1325] [Q&A / Help]
Hi, Thanks for excellent work.  

Can we change multiple items like dragging them and changing the labels into specific label? 
I have A, B, C, D .... N(like 100 classes) on labeled data, and I need to change them into "Z" at once.

If you have any ideas, please let me know.

**Comment by Ynjxsjmh** [comment]
IMO, you can use regex or something to handle the json file directly.
